cmd/e2e/main.go

```diff
@@ -10,7 +10,7 @@ import (
 	"time"
 
 	"github.com/google/uuid"
-	mpsv1alpha1 "github.com/playfab/thundernetes/operator/api/v1alpha1"
+	mpsv1alpha1 "github.com/playfab/thundernetes/pkg/operator/api/v1alpha1"
```

Comment:
currently, the go.mod on e2e tests has a replace command so that this folder is referenced locally. How does this work now? Are we sure that this command will not try and fetch the api/v1alpha1 from upstream? I probably miss something on the way Go modules work, just wanna make sure

Reply:
since this path is a subpath of the existing module, it references it locally